statchatui/src/Compare/Compare.jsx

```diff
@@ -21,10 +21,53 @@ const Compare = ({ signedInUserData }) => {
   const [isLoading, setIsLoading] = useState(false);
   const [otherUserStats, setOtherUserStats] = useState(null);
   const [signedInUserStats, setSignedInUserStats] = useState(null);
+  const [userQueue, setUserQueue] = useState([]);
+  const [showStats, setShowStats] = useState(null);
+  const storedUserQueue = sessionStorage.getItem("userQueue");
+
+  //get the previously stored userqueue everytime the page mounts
+  useEffect(() => {
+    if (storedUserQueue) {
+      setUserQueue(JSON.parse(storedUserQueue));
+    }
+  }, []);
```

Comment:
You can simplify all of this to:

```
const storedUserQueue = sessionStorage.getItem("userQueue");
const [userQueue, setUserQueue] = useState(storedUserQueue);
```

The first parameter in `useState` is the initial value, which is what you want here!